File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l = 0
        r = len(numbers) - 1
        sum = numbers[l] + numbers[r]
        while sum != target:
            if sum < target:
                l += 1
            else:
                r -= 1

            sum = numbers[l] + numbers[r]

        return [l + 1, r + 1]

File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
    let l = 0;
    let r = numbers.length - 1;
    let sum = numbers[l] + numbers[r];
    while(sum !== target){
        if(sum > target){
            r -= 1;
        }else{
            l += 1
        }
        sum = numbers[l] + numbers[r]
    }

    return [l + 1,r + 1]
};
